### DISCUSSION TITLE
Create new side menu option in pimcore?

### DISCUSSION THREAD

**Discussion by Dizzyggs** [pimcore/pimcore#13784] [Q&A / Support]
Hello. I can't for the life of me find any documentation on how to go ahead and create a new menu in the side navigation? 

Can anyone referr me to any documentation? Thank you!

**Comment by alexmf23** [comment]
Hey this might help
https://pimcore.com/docs/pimcore/current/Development_Documentation/Extending_Pimcore/Bundle_Developers_Guide/Event_Listener_UI.html

**Comment by dpfaffenbauer** [comment]
CoreShop's Menu Bundle allows menus to be defined in PHP with Permissions checks in PHP too: https://docs.coreshop.org/3.0.0/Bundles/Menu_Bundle.html
